bucket/bruno.json

```
@@ -0,0 +1,29 @@
+{
+    "version": "0.25.0",
+    "homepage": "https://www.usebruno.com/",
+    "license": "MIT",
+    "description": "Opensource IDE For Exploring and Testing Api's (lightweight alternative to postman/insomnia)",
```

```suggestion
    "description": "Open source IDE for exploring and testing APIs (lightweight alternative to Postman/Insomnia)",
```
Improvement to spelling in description